tidy3d/__init__.py

```diff
@@ -114,3 +114,147 @@ def set_logging_level(level: str) -> None:
+    "Medium",
+    "PoleResidue",
+    "AnisotropicMedium",
+    "PEC",
+    "PECMedium",
+    "Medium2D",
+    "Sellmeier",
+    "Debye",
+    "Drude",
+    "Lorentz",
+    "CustomMedium",
+    "CustomPoleResidue",
+    "CustomSellmeier",
+    "FullyAnisotropicMedium",
+    "CustomLorentz",
+    "CustomDrude",
+    "CustomDebye",
+    "CustomAnisotropicMedium",
+    "RotationAroundAxis",
+    "PerturbationMedium",
+    "PerturbationPoleResidue",
+    "ParameterPerturbation",
+    "LinearHeatPerturbation",
+    "CustomHeatPerturbation",
+    "LinearChargePerturbation",
+    "CustomChargePerturbation",
+    "Structure",
+    "MeshOverrideStructure",
+    "ModeSpec",
+    "ApodizationSpec",
+    "GaussianPulse",
+    "ContinuousWave",
+    "CustomSourceTime",
+    "UniformCurrentSource",
+    "PlaneWave",
+    "ModeSource",
+    "PointDipole",
+    "GaussianBeam",
+    "AstigmaticGaussianBeam",
+    "CustomFieldSource",
+    "TFSF",
+    "CustomCurrentSource",
+    "FieldMonitor",
+    "FieldTimeMonitor",
+    "FluxMonitor",
+    "FluxTimeMonitor",
+    "ModeMonitor",
+    "ModeSolverMonitor",
+    "PermittivityMonitor",
+    "FieldProjectionAngleMonitor",
+    "FieldProjectionCartesianMonitor",
+    "FieldProjectionKSpaceMonitor",
+    "FieldProjectionSurface",
+    "DiffractionMonitor",
+    "Simulation",
+    "FieldProjector",
+    "ScalarFieldDataArray",
+    "ScalarModeFieldDataArray",
+    "ScalarFieldTimeDataArray",
+    "SpatialDataArray",
+    "ModeAmpsDataArray",
+    "ModeIndexDataArray",
+    "FluxDataArray",
+    "FluxTimeDataArray",
+    "FieldProjectionAngleDataArray",
+    "FieldProjectionCartesianDataArray",
+    "FieldProjectionKSpaceDataArray",
+    "DiffractionDataArray",
+    "HeatDataArray",
+    "ChargeDataArray",
+    "FieldDataset",
+    "FieldTimeDataset",
+    "PermittivityDataset",
+    "ModeSolverDataset",
+    "FieldData",
+    "FieldTimeData",
+    "PermittivityData",
+    "FluxData",
+    "FluxTimeData",
+    "ModeData",
+    "ModeSolverData",
+    "AbstractFieldProjectionData",
+    "FieldProjectionAngleData",
+    "FieldProjectionCartesianData",
+    "FieldProjectionKSpaceData",
+    "DiffractionData",
+    "SimulationData",
+    "DATA_TYPE_MAP",
+    "BoundarySpec",
+    "Boundary",
+    "BoundaryEdge",
+    "BoundaryEdgeType",
+    "BlochBoundary",
+    "Periodic",
+    "PECBoundary",
+    "PMCBoundary",
+    "PML",
+    "StablePML",
+    "Absorber",
+    "PMLParams",
+    "AbsorberParams",
+    "PMLTypes",
+    "DefaultPMLParameters",
+    "DefaultStablePMLParameters",
+    "DefaultAbsorberParameters",
+    "C_0",
+    "ETA_0",
+    "HBAR",
+    "EPSILON_0",
+    "MU_0",
+    "Q_e",
+    "K_B",
+    "inf",
+    "material_library",
+    "Graphene",
+    "AbstractMedium",
+    "Geometry",
+    "Source",
+    "SourceTime",
+    "Monitor",
+    "YeeGrid",
+    "FieldGrid",
+    "Coords1D",
+    "log",
+    "set_logging_file",
+    "set_logging_console",
+    "config",
+    "__version__",
+    "Updater",
+]
```

Comment:
what is this about?

Reply:
something ruff was complaining about. __all__ controls what gets imported when you import * from the module. Guess it's good practice to include it